.github/workflows/stale.yml

```diff
@@ -0,0 +1,33 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It would also be great if we could also leverage the need-information label
auto closing issues with at label after perhaps only 14 days. 150 open issues in the repo have that label.
 stale-issue-label: "status/need-information"

example of a workflow like this:
https://github.com/dart-lang/dart-pad/blob/290442b262bc28b190c11bc7b23b13b2636f149a/.github/workflows/no-response.yml